### PR TITLE
nvidia-settings app

### PR DESCRIPTION
**Pull request by adityashah1212**

### CODE REVIEW

File: nvidia-settings.json.in [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+					"path":"nvidia-settings.desktop"
+				},
+				{
+					"type":"file",
+					"path":"nvidia-settings.png"
+				}
+			],
+			"modules":[
+				{
+					"name":"gtk",
```
Comment (TingPing) [inline nvidia-settings.json.in]: IIRC you can compile against Gtk3.

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```
Comment (TingPing) [inline build.sh]: Can't use a custom build script on flathub. Do you actually need a version for each driver?

File: nvidia-settings.json.in (hunk 1 above)
Comment (adityashah1212) [inline nvidia-settings.json.in]: I tried it didn't work, initially it warns latter it fails. I will try deleting everything from gtk2 build and see what happens

**Comment by TingPing** [comment]
I'm not actually sure this should be in flathub at all. It seems like a core component that should be installed on the host alongside the driver itself.

File: build.sh (hunk 2 above)
Comment (adityashah1212) [inline build.sh]: I tried 384.47 with 381.22, it worked. Though I am not sure if that is the case with all versions

**Comment by adityashah1212** [comment]
Well I am not sure about it either. Since I use it to only monitor GPU temperature once in a while. Never used the powermizer though. Beyond that I don't see much of its use.

**Comment by TingPing** [comment]
I'll just be closing this, as mentioned driver utilities are not the type of application intended for flathub, maybe this should be discussed on the freedesktop Sdk where nvidia drivers are built.

**Pull request closed without merging**